.env.default

```diff
@@ -6,5 +6,8 @@ STRIPE_PUBLIC=pk_test_51HERaHFHEvC1s7vkxMfTOWuGCHWCyukcRbwF5WNOUgeWWkAUHj2btlzxG
 
 # Request the following values from support
 # CLERK_SECRET_KEY
-# STRIPE_SECRET
+# STRIPE_SECRET_KEY
 # STRIPE_WEBHOOK_SECRET
+
+# Uncomment the following line to enable debug logging
+# DEBUG=api,sendmessage
```

Comment:
Let's actually move this to the contribution docs.

Mac users will be able to just do `DEBUG={keys} npm start`, and set of supported `{keys}` include more than just `api` and `sendmessage`